From 0d79bc28a68e8ad7b4feed3e920607906074d98f Mon Sep 17 00:00:00 2001
From: liuzhensong <liuzhensong@loongson.cn>
Date: Thu, 12 Aug 2021 16:21:09 +0800
Subject: [PATCH 19/30] binutils: change copyright time.

  bfd/cpu-loongarch.c
  bfd/elfnn-loongarch.c
  bfd/elfxx-loongarch.c
  bfd/elfxx-loongarch.h
  gas/config/loongarch-lex-wrapper.c
  gas/config/loongarch-lex.l
  gas/config/loongarch-parse.y
  gas/config/tc-loongarch.c
  gas/config/tc-loongarch.h
  gas/doc/c-loongarch.texi
  gas/testsuite/gas/loongarch/loongarch.exp
  gdb/arch/loongarch-linux-nat.c
  gdb/arch/loongarch-linux-nat.h
  gdb/arch/loongarch.c
  gdb/arch/loongarch.h
  gdb/features/loongarch/base32.xml
  gdb/features/loongarch/base64.xml
  gdb/features/loongarch/fpu32.xml
  gdb/features/loongarch/fpu64.xml
  gdb/features/loongarch/lasx.xml
  gdb/features/loongarch/lbt32.xml
  gdb/features/loongarch/lbt64.xml
  gdb/features/loongarch/lsx.xml
  gdb/loongarch-linux-nat.c
  gdb/loongarch-linux-tdep.c
  gdb/loongarch-linux-tdep.h
  gdb/loongarch-tdep.c
---
 bfd/cpu-loongarch.c                       | 2 +-
 bfd/elfnn-loongarch.c                     | 2 +-
 bfd/elfxx-loongarch.c                     | 2 +-
 bfd/elfxx-loongarch.h                     | 2 +-
 gas/config/loongarch-lex-wrapper.c        | 2 +-
 gas/config/loongarch-lex.l                | 2 +-
 gas/config/loongarch-parse.y              | 2 +-
 gas/config/tc-loongarch.c                 | 2 +-
 gas/config/tc-loongarch.h                 | 2 +-
 gas/doc/c-loongarch.texi                  | 2 +-
 gas/testsuite/gas/loongarch/loongarch.exp | 2 +-
 include/elf/loongarch.h                   | 2 +-
 include/opcode/loongarch.h                | 2 +-
 ld/emultempl/loongarchelf.em              | 2 +-
 opcodes/loongarch-coder.c                 | 2 +-
 opcodes/loongarch-dis.c                   | 2 +-
 opcodes/loongarch-opc.c                   | 2 +-
 17 files changed, 17 insertions(+), 17 deletions(-)

diff --git a/bfd/cpu-loongarch.c b/bfd/cpu-loongarch.c
index 0502113e04b..25704d4749c 100644
--- a/bfd/cpu-loongarch.c
+++ b/bfd/cpu-loongarch.c
@@ -1,5 +1,5 @@
 /* BFD support for Loongarch.
-   Copyright (C) 2021-2021 Free Software Foundation, Inc.
+   Copyright (C) 2021 Free Software Foundation, Inc.
    Contributed by Loongson Ltd.
 
    This file is part of BFD, the Binary File Descriptor library.
diff --git a/bfd/elfnn-loongarch.c b/bfd/elfnn-loongarch.c
index 2c56b7b1462..c8a828d74d1 100644
--- a/bfd/elfnn-loongarch.c
+++ b/bfd/elfnn-loongarch.c
@@ -1,5 +1,5 @@
 /* Loongarch-specific support for NN-bit ELF.
-   Copyright (C) 2021-2021 Free Software Foundation, Inc.
+   Copyright (C) 2021 Free Software Foundation, Inc.
    Contributed by Loongson Ltd.
 
    This file is part of BFD, the Binary File Descriptor library.
diff --git a/bfd/elfxx-loongarch.c b/bfd/elfxx-loongarch.c
index 7fc46de8854..870b38f1dac 100644
--- a/bfd/elfxx-loongarch.c
+++ b/bfd/elfxx-loongarch.c
@@ -1,5 +1,5 @@
 /* Loongarch-specific support for ELF.
-   Copyright (C) 2021-2021 Free Software Foundation, Inc.
+   Copyright (C) 2021 Free Software Foundation, Inc.
    Contributed by Loongson Ltd.
 
    This file is part of BFD, the Binary File Descriptor library.
diff --git a/bfd/elfxx-loongarch.h b/bfd/elfxx-loongarch.h
index 375f1827013..0cf8ae442e2 100644
--- a/bfd/elfxx-loongarch.h
+++ b/bfd/elfxx-loongarch.h
@@ -1,5 +1,5 @@
 /* Loonarch-specific backend routines.
-   Copyright (C) 2021-2021 Free Software Foundation, Inc.
+   Copyright (C) 2021 Free Software Foundation, Inc.
    Contributed by Loongson Ltd.
 
    This file is part of BFD, the Binary File Descriptor library.
diff --git a/gas/config/loongarch-lex-wrapper.c b/gas/config/loongarch-lex-wrapper.c
index 2d7509b1b20..5a13a6e861a 100644
--- a/gas/config/loongarch-lex-wrapper.c
+++ b/gas/config/loongarch-lex-wrapper.c
@@ -1,4 +1,4 @@
-/* Copyright (C) 2021-2021 Free Software Foundation, Inc.
+/* Copyright (C) 2021 Free Software Foundation, Inc.
 
    This file is part of GAS, the GNU Assembler.
 
diff --git a/gas/config/loongarch-lex.l b/gas/config/loongarch-lex.l
index b6397726e79..1344a288065 100644
--- a/gas/config/loongarch-lex.l
+++ b/gas/config/loongarch-lex.l
@@ -1,6 +1,6 @@
 %option noyywrap
 /*
-   Copyright (C) 2021-2021 Free Software Foundation, Inc.
+   Copyright (C) 2021 Free Software Foundation, Inc.
 
    This file is part of GAS, the GNU Assembler.
 
diff --git a/gas/config/loongarch-parse.y b/gas/config/loongarch-parse.y
index 99929a26c4d..843d0feb473 100644
--- a/gas/config/loongarch-parse.y
+++ b/gas/config/loongarch-parse.y
@@ -1,5 +1,5 @@
 /*
-   Copyright (C) 2021-2021 Free Software Foundation, Inc.
+   Copyright (C) 2021 Free Software Foundation, Inc.
 
    This file is part of GAS, the GNU Assembler.
 
diff --git a/gas/config/tc-loongarch.c b/gas/config/tc-loongarch.c
index 94ae2fb3b32..1589c6f6acd 100644
--- a/gas/config/tc-loongarch.c
+++ b/gas/config/tc-loongarch.c
@@ -1,6 +1,6 @@
 /* tc-loongarch.c -- Assemble for the Loongarch ISA
 
-   Copyright (C) 2021-2021 Free Software Foundation, Inc.
+   Copyright (C) 2021 Free Software Foundation, Inc.
    Contributed by Loongson Ltd.
 
    This file is part of GAS.
diff --git a/gas/config/tc-loongarch.h b/gas/config/tc-loongarch.h
index f8e76cee5cf..f1917526da6 100644
--- a/gas/config/tc-loongarch.h
+++ b/gas/config/tc-loongarch.h
@@ -1,5 +1,5 @@
 /* tc-loongarch.h -- Header file for tc-loongarch.c.
-   Copyright (C) 2021-2021 Free Software Foundation, Inc.
+   Copyright (C) 2021 Free Software Foundation, Inc.
    Contributed by Loongson Ltd.
 
    This file is part of GAS.
diff --git a/gas/doc/c-loongarch.texi b/gas/doc/c-loongarch.texi
index cdadec528f4..2a139484577 100644
--- a/gas/doc/c-loongarch.texi
+++ b/gas/doc/c-loongarch.texi
@@ -1,4 +1,4 @@
-@c Copyright (C) 2021-2021 Free Software Foundation, Inc.
+@c Copyright (C) 2021 Free Software Foundation, Inc.
 @c This is part of the GAS anual.
 @c For copying conditions, see the file as.texinfo
 @c man end
diff --git a/gas/testsuite/gas/loongarch/loongarch.exp b/gas/testsuite/gas/loongarch/loongarch.exp
index 1b3240870df..c186b67315f 100644
--- a/gas/testsuite/gas/loongarch/loongarch.exp
+++ b/gas/testsuite/gas/loongarch/loongarch.exp
@@ -1,5 +1,5 @@
 # Expect script for LoongArch assembler tests.
-#   Copyright (C) 2021-2021 Free Software Foundation, Inc.
+#   Copyright (C) 2021 Free Software Foundation, Inc.
 #
 # This file is part of the GNU Binutils.
 #
diff --git a/include/elf/loongarch.h b/include/elf/loongarch.h
index 91e8c4e53ae..3289bc87545 100644
--- a/include/elf/loongarch.h
+++ b/include/elf/loongarch.h
@@ -1,4 +1,4 @@
-/* Copyright (C) 2021-2021 Free Software Foundation, Inc.
+/* Copyright (C) 2021 Free Software Foundation, Inc.
    Contributed by Loongson Ltd.
 
    This file is part of GNU Binutils.
diff --git a/include/opcode/loongarch.h b/include/opcode/loongarch.h
index b7fe8dd8373..4b93762230e 100644
--- a/include/opcode/loongarch.h
+++ b/include/opcode/loongarch.h
@@ -1,6 +1,6 @@
 /* Loongarch assembler/disassembler support.
 
-   Copyright (C) 2021-2021 Free Software Foundation, Inc.
+   Copyright (C) 2021 Free Software Foundation, Inc.
    Contributed by Loongson Ltd.
 
    This file is part of GNU Binutils.
diff --git a/ld/emultempl/loongarchelf.em b/ld/emultempl/loongarchelf.em
index 8751a628306..b688ef7bc1d 100644
--- a/ld/emultempl/loongarchelf.em
+++ b/ld/emultempl/loongarchelf.em
@@ -1,5 +1,5 @@
 # This shell script emits a C file. -*- C -*-
-#   Copyright (C) 2021-2021 Free Software Foundation, Inc.
+#   Copyright (C) 2021 Free Software Foundation, Inc.
 #   Contributed by Loongson Ltd.
 #
 # This file is part of the GNU Binutils.
diff --git a/opcodes/loongarch-coder.c b/opcodes/loongarch-coder.c
index 9e3c45dc7bc..2b042fd9866 100644
--- a/opcodes/loongarch-coder.c
+++ b/opcodes/loongarch-coder.c
@@ -1,5 +1,5 @@
 /* loongarch opcode support.
-   Copyright (C) 2021-2021 Free Software Foundation, Inc.
+   Copyright (C) 2021 Free Software Foundation, Inc.
    Contributed by Loongson Ltd.
 
    This file is part of the GNU opcodes library.
diff --git a/opcodes/loongarch-dis.c b/opcodes/loongarch-dis.c
index 0bce27c038e..3a3ea1b1f6c 100644
--- a/opcodes/loongarch-dis.c
+++ b/opcodes/loongarch-dis.c
@@ -1,5 +1,5 @@
 /* loongarch opcode support.
-   Copyright (C) 2021-2021 Free Software Foundation, Inc.
+   Copyright (C) 2021 Free Software Foundation, Inc.
    Contributed by Loongson Ltd.
 
    This file is part of the GNU opcodes library.
diff --git a/opcodes/loongarch-opc.c b/opcodes/loongarch-opc.c
index c24fa65f1d1..1eb2a711201 100644
--- a/opcodes/loongarch-opc.c
+++ b/opcodes/loongarch-opc.c
@@ -1,5 +1,5 @@
 /* loongarch opcode support.
-   Copyright (C) 2021-2021 Free Software Foundation, Inc.
+   Copyright (C) 2021 Free Software Foundation, Inc.
    Contributed by Loongson Ltd.
 
    This file is part of the GNU opcodes library.
-- 
2.30.1

